File: flutter_app/lib/models/task.dart
class Task {
  final String id;
  final String title;
  final String? description;
  final String? assignedTo;
  final String status;
  final String priority;
  final String? dueDate;
  final bool isPqcVerified;
  final String? pqcSignature;

  Task({
    required this.id,
    required this.title,
    this.description,
    this.assignedTo,
    required this.status,
    required this.priority,
    this.dueDate,
    this.isPqcVerified = false,
    this.pqcSignature,
  });

  factory Task.fromJson(
    Map<String, dynamic> json
  ) {
    return Task(
      id: json['id']?.toString() ?? '',
      title: json['title']
        ?.toString() ?? '',
      description: json['description']
        ?.toString(),
      assignedTo: json['assigned_to']
        ?.toString(),
      status: json['status']
        ?.toString() ?? 'pending',
      priority: json['priority']
        ?.toString() ?? 'medium',
      dueDate: json['due_date']
        ?.toString(),
      isPqcVerified: 
        json['is_pqc_verified'] ?? false,
      pqcSignature: json['pqc_signature']
        ?.toString(),
    );
  }
}

File: backend/models/task.py
from pydantic import BaseModel
from typing import Optional
from datetime import datetime
from uuid import UUID

class TaskBase(BaseModel):
    title: str
    description: Optional[str] = None
    assigned_to: Optional[str] = None
    priority: str = "medium"
    due_date: Optional[str] = None

class TaskCreate(TaskBase):
    pass

class TaskUpdate(BaseModel):
    status: Optional[str] = None
    priority: Optional[str] = None
    due_date: Optional[datetime] = None

class TaskResponse(TaskBase):
    id: UUID
    status: str
    created_at: datetime
    
    class Config:
        from_attributes = True
